integrations/db/mysql/pom.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Copyright (c) 2020 Oracle and/or its affiliates.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.helidon.integrations.db</groupId>
+        <artifactId>helidon-integrations-db-project</artifactId>
+        <version>2.1.1-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>mysql</artifactId>
```

Comment:
The artifactId is wrong, see https://github.com/oracle/helidon/blob/master/DEV-GUIDELINES.md#package-and-module-structure

Reply:
I was not 1st one who started implementing modules in this section so I just followed conventions that were there before. :)

Reply:
Anyway, parent project artifact id is helidon-integrations-db-project so renaming my modules to
- helidon-integrations-db-mysql
- helidon-integrations-db-mariadb
- helidon-integrations-db-pgsql
I did not touch H2 and Oracle modules. They shall be fixed in a separate PR.